instrumentation/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/baggage/BaggageBridging.java

```diff
@@ -12,7 +12,7 @@
 public final class BaggageBridging {
 
   public static Baggage toApplication(io.opentelemetry.api.baggage.Baggage agentBaggage) {
-    BaggageBuilder applicationBaggageBuilder = Baggage.builder().setNoParent();
+    BaggageBuilder applicationBaggageBuilder = Baggage.builder();
```

Comment:
this is better too 👍

Reply:
Yeah I was quite happy with how baggage turned out here and in the SDK's unit tests too, much simpler API.